Many GCC warnings due to FBS

### Your environment

- Operating system: Debian GNU/Linux 11
- gcc/clang version: gcc version 10.2.1 20210110 (Debian 10.2.1-6)
- mediasoup version: flatbuffers branch
- mediasoup-client version:


```
[1073/1172] Compiling C++ object mediasoup-worker.p/src_RTC_IceCandidate.cpp.o
../../../src/RTC/IceCandidate.cpp: In static member function ‘static RTC::IceCandidate::CandidateType RTC::IceCandidate::CandidateTypeFromFbs(FBS::WebRtcTransport::IceCandidateType)’:
../../../src/RTC/IceCandidate.cpp:19:2: warning: control reaches end of non-void function [-Wreturn-type]
   19 |  }
      |  ^
../../../src/RTC/IceCandidate.cpp: In static member function ‘static FBS::WebRtcTransport::IceCandidateType RTC::IceCandidate::CandidateTypeToFbs(RTC::IceCandidate::CandidateType)’:
../../../src/RTC/IceCandidate.cpp:28:2: warning: control reaches end of non-void function [-Wreturn-type]
   28 |  }
      |  ^
../../../src/RTC/IceCandidate.cpp: In static member function ‘static RTC::IceCandidate::TcpCandidateType RTC::IceCandidate::TcpCandidateTypeFromFbs(FBS::WebRtcTransport::IceCandidateTcpType)’:
../../../src/RTC/IceCandidate.cpp:38:2: warning: control reaches end of non-void function [-Wreturn-type]
   38 |  }
      |  ^
../../../src/RTC/IceCandidate.cpp: In static member function ‘static FBS::WebRtcTransport::IceCandidateTcpType RTC::IceCandidate::TcpCandidateTypeToFbs(RTC::IceCandidate::TcpCandidateType)’:
../../../src/RTC/IceCandidate.cpp:48:2: warning: control reaches end of non-void function [-Wreturn-type]
   48 |  }
      |  ^
[1074/1172] Compiling C++ object mediasoup-worker.p/src_RTC_IceServer.cpp.o
../../../src/RTC/IceServer.cpp: In function ‘RTC::IceServer::IceState RTC::IceStateFromFbs(FBS::WebRtcTransport::IceState)’:
../../../src/RTC/IceServer.cpp:34:2: warning: control reaches end of non-void function [-Wreturn-type]
   34 |  }
      |  ^
../../../src/RTC/IceServer.cpp: In static member function ‘static FBS::WebRtcTransport::IceState RTC::IceServer::IceStateToFbs(RTC::IceServer::IceState)’:
../../../src/RTC/IceServer.cpp:52:2: warning: control reaches end of non-void function [-Wreturn-type]
   52 |  }
      |  ^
[1083/1172] Compiling C++ object mediasoup-worker.p/src_RTC_Producer.cpp.o
In file included from ../../../src/RTC/Producer.cpp:9:
../../../include/RTC/Codecs/Tools.hpp: In static member function ‘static bool RTC::Codecs::Tools::IsValidTypeForCodec(RTC::RtpParameters::Type, const RTC::RtpCodecMimeType&)’:
../../../include/RTC/Codecs/Tools.hpp:165:4: warning: control reaches end of non-void function [-Wreturn-type]
  165 |    }
      |    ^
[1088/1172] Compiling C++ object mediasoup-worker.p/src_RTC_RtpPacket.cpp.o
In file included from ../../../subprojects/flatbuffers-23.3.3/include/flatbuffers/flatbuffer_builder.h:35,
                 from ../../../subprojects/flatbuffers-23.3.3/include/flatbuffers/flatbuffers.h:29,
                 from fbs/FBS/rtpPacket.h:7,
                 from ../../../include/RTC/RtpPacket.hpp:6,
                 from ../../../src/RTC/RtpPacket.cpp:4:
../../../subprojects/flatbuffers-23.3.3/include/flatbuffers/vector_downward.h: In member function ‘flatbuffers::Offset<FBS::RtpPacket::Dump> RTC::RtpPacket::FillBuffer(flatbuffers::FlatBufferBuilder&) const’:
../../../subprojects/flatbuffers-23.3.3/include/flatbuffers/vector_downward.h:190:34: warning: ‘wideSequenceNumber’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  190 |     *reinterpret_cast<T *>(cur_) = little_endian_t;
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
../../../src/RTC/RtpPacket.cpp:348:12: note: ‘wideSequenceNumber’ was declared here
  348 |   uint16_t wideSequenceNumber;
      |            ^~~~~~~~~~~~~~~~~~
[1091/1172] Compiling C++ object mediasoup-worker.p/src_RTC_RtpStreamSend.cpp.o
../../../src/RTC/RtpStreamSend.cpp: In constructor ‘RTC::RtpStreamSend::RtpStreamSend(RTC::RtpStreamSend::Listener*, RTC::RtpStream::Params&, std::string&)’:
../../../src/RTC/RtpStreamSend.cpp:57:78: warning: ‘maxRetransmissionDelayMs’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   57 |      RetransmissionBufferMaxItems, maxRetransmissionDelayMs, params.clockRate);
      |                                                                              ^
[1099/1172] Compiling C++ object mediasoup-worker.p/src_RTC_SrtpSession.cpp.o
../../../src/RTC/SrtpSession.cpp: In static member function ‘static FBS::SrtpParameters::SrtpCryptoSuite RTC::SrtpSession::CryptoSuiteToFbs(RTC::SrtpSession::CryptoSuite)’:
../../../src/RTC/SrtpSession.cpp:47:2: warning: control reaches end of non-void function [-Wreturn-type]
   47 |  }
      |  ^
../../../src/RTC/SrtpSession.cpp: In static member function ‘static RTC::SrtpSession::CryptoSuite RTC::SrtpSession::CryptoSuiteFromFbs(FBS::SrtpParameters::SrtpCryptoSuite)’:
../../../src/RTC/SrtpSession.cpp:65:2: warning: control reaches end of non-void function [-Wreturn-type]
   65 |  }
      |  ^
[1103/1172] Compiling C++ object mediasoup-worker.p/src_RTC_SimulcastConsumer.cpp.o
In file included from ../../../src/RTC/SimulcastConsumer.cpp:9:
../../../include/RTC/Codecs/Tools.hpp: In static member function ‘static bool RTC::Codecs::Tools::IsValidTypeForCodec(RTC::RtpParameters::Type, const RTC::RtpCodecMimeType&)’:
../../../include/RTC/Codecs/Tools.hpp:165:4: warning: control reaches end of non-void function [-Wreturn-type]
  165 |    }
      |    ^
[1105/1172] Compiling C++ object mediasoup-worker.p/src_RTC_SvcConsumer.cpp.o
In file included from ../../../src/RTC/SvcConsumer.cpp:9:
../../../include/RTC/Codecs/Tools.hpp: In static member function ‘static bool RTC::Codecs::Tools::IsValidTypeForCodec(RTC::RtpParameters::Type, const RTC::RtpCodecMimeType&)’:
../../../include/RTC/Codecs/Tools.hpp:165:4: warning: control reaches end of non-void function [-Wreturn-type]
  165 |    }
      |    ^
[1107/1172] Compiling C++ object mediasoup-worker.p/src_RTC_TransportTuple.cpp.o
../../../src/RTC/TransportTuple.cpp: In static member function ‘static RTC::TransportTuple::Protocol RTC::TransportTuple::ProtocolFromFbs(FBS::Transport::Protocol)’:
../../../src/RTC/TransportTuple.cpp:22:2: warning: control reaches end of non-void function [-Wreturn-type]
   22 |  }
      |  ^
../../../src/RTC/TransportTuple.cpp: In static member function ‘static FBS::Transport::Protocol RTC::TransportTuple::ProtocolToFbs(RTC::TransportTuple::Protocol)’:
../../../src/RTC/TransportTuple.cpp:34:2: warning: control reaches end of non-void function [-Wreturn-type]
   34 |  }
      |  ^
[1115/1172] Compiling C++ object mediasoup-worker.p/src_RTC_Transport.cpp.o
In file included from fbs/FBS/directTransport.h:16,
                 from fbs/FBS/notification.h:20,
                 from fbs/FBS/message.h:17,
                 from ../../../include/Channel/ChannelRequest.hpp:5,
                 from ../../../include/Settings.hpp:6,
                 from ../../../include/Logger.hpp:91,
                 from ../../../src/RTC/Transport.cpp:5:
fbs/FBS/transport.h: In member function ‘flatbuffers::Offset<FBS::Transport::Stats> RTC::Transport::FillBufferStats(flatbuffers::FlatBufferBuilder&)’:
fbs/FBS/transport.h:2012:29: warning: ‘sctpState’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 2012 |     fbb_.AddElement<uint8_t>(Stats::VT_SCTPSTATE, static_cast<uint8_t>(sctpState));
      |     ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../../../src/RTC/Transport.cpp:398:35: note: ‘sctpState’ was declared here
  398 |   FBS::SctpAssociation::SctpState sctpState;
      |                                   ^~~~~~~~~
[1123/1172] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpHeaderExtensionUri.cpp.o
../../../src/RTC/RtpDictionaries/RtpHeaderExtensionUri.cpp: In static member function ‘static RTC::RtpHeaderExtensionUri::Type RTC::RtpHeaderExtensionUri::TypeFromFbs(FBS::RtpParameters::RtpHeaderExtensionUri)’:
../../../src/RTC/RtpDictionaries/RtpHeaderExtensionUri.cpp:40:2: warning: control reaches end of non-void function [-Wreturn-type]
   40 |  }
      |  ^
../../../src/RTC/RtpDictionaries/RtpHeaderExtensionUri.cpp: In static member function ‘static FBS::RtpParameters::RtpHeaderExtensionUri RTC::RtpHeaderExtensionUri::TypeToFbs(RTC::RtpHeaderExtensionUri::Type)’:
../../../src/RTC/RtpDictionaries/RtpHeaderExtensionUri.cpp:70:2: warning: control reaches end of non-void function [-Wreturn-type]
   70 |  }
      |  ^
[1129/1172] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelNotification.cpp.o
In file included from ../../../src/Channel/ChannelNotification.cpp:5:
../../../src/Channel/ChannelNotification.cpp: In constructor ‘Channel::ChannelNotification::ChannelNotification(const FBS::Notification::Notification*)’:
../../../include/Logger.hpp:397:80: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 6 has type ‘Channel::ChannelNotification::Event’ {aka ‘FBS::Notification::Event’} [-Wformat=]
  397 |    const int loggerWritten = std::snprintf(Logger::buffer, Logger::bufferSize, "E" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
../../../include/MediaSoupErrors.hpp:32:3: note: in expansion of macro ‘MS_ERROR’
   32 |   MS_ERROR("throwing MediaSoupError: " desc, ##__VA_ARGS__); \
      |   ^~~~~~~~
../../../src/Channel/ChannelNotification.cpp:33:4: note: in expansion of macro ‘MS_THROW_ERROR’
   33 |    MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
      |    ^~~~~~~~~~~~~~
In file included from ../../../src/Channel/ChannelNotification.cpp:6:
../../../src/Channel/ChannelNotification.cpp:33:19: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 4 has type ‘Channel::ChannelNotification::Event’ {aka ‘FBS::Notification::Event’} [-Wformat=]
   33 |    MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~~~~~~
      |                                                       |
      |                                                       Channel::ChannelNotification::Event {aka FBS::Notification::Event}
../../../src/Channel/ChannelNotification.cpp:33:4: note: in expansion of macro ‘MS_THROW_ERROR’
   33 |    MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
      |    ^~~~~~~~~~~~~~
[1130/1172] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelRequest.cpp.o
In file included from ../../../src/Channel/ChannelRequest.cpp:5:
../../../src/Channel/ChannelRequest.cpp: In constructor ‘Channel::ChannelRequest::ChannelRequest(Channel::ChannelSocket*, const FBS::Request::Request*)’:
../../../include/Logger.hpp:397:80: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 6 has type ‘Channel::ChannelRequest::Method’ {aka ‘FBS::Request::Method’} [-Wformat=]
  397 |    const int loggerWritten = std::snprintf(Logger::buffer, Logger::bufferSize, "E" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
../../../include/MediaSoupErrors.hpp:32:3: note: in expansion of macro ‘MS_ERROR’
   32 |   MS_ERROR("throwing MediaSoupError: " desc, ##__VA_ARGS__); \
      |   ^~~~~~~~
../../../src/Channel/ChannelRequest.cpp:105:4: note: in expansion of macro ‘MS_THROW_ERROR’
  105 |    MS_THROW_ERROR("unknown method '%" PRIu8 "'", this->method);
      |    ^~~~~~~~~~~~~~
In file included from ../../../src/Channel/ChannelRequest.cpp:6:
../../../src/Channel/ChannelRequest.cpp:105:19: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 4 has type ‘Channel::ChannelRequest::Method’ {aka ‘FBS::Request::Method’} [-Wformat=]
  105 |    MS_THROW_ERROR("unknown method '%" PRIu8 "'", this->method);
      |                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ~~~~~~~~~~~~
      |                                                        |
      |                                                        Channel::ChannelRequest::Method {aka FBS::Request::Method}
../../../include/MediaSoupErrors.hpp:33:69: note: in definition of macro ‘MS_THROW_ERROR’
   33 |   std::snprintf(MediaSoupError::buffer, MediaSoupError::bufferSize, desc, ##__VA_ARGS__); \
      |                                                                     ^~~~
[1131/1172] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelSocket.cpp.o
../../../src/Channel/ChannelSocket.cpp: In member function ‘virtual void Channel::ChannelSocket::OnConsumerSocketMessage(Channel::ConsumerSocket*, char*, size_t)’:
../../../src/Channel/ChannelSocket.cpp:306:11: warning: ‘notification’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  306 |    delete notification;
      |           ^~~~~~~~~~~~
../../../src/Channel/ChannelSocket.cpp:281:23: warning: ‘request’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  281 |     request->TypeError(error.what());
      |     ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
../../../src/Channel/ChannelSocket.cpp: In member function ‘bool Channel::ChannelSocket::CallbackRead()’:
../../../src/Channel/ChannelSocket.cpp:226:12: warning: ‘notification’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  226 |     delete notification;
      |            ^~~~~~~~~~~~
../../../src/Channel/ChannelSocket.cpp:201:24: warning: ‘request’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  201 |      request->TypeError(error.what());
      |      ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
[1138/1172] Compiling C++ object mediasoup-worker.p/src_RTC_DtlsTransport.cpp.o
../../../src/RTC/DtlsTransport.cpp: In static member function ‘static RTC::DtlsTransport::Role RTC::DtlsTransport::RoleFromFbs(FBS::WebRtcTransport::DtlsRole)’:
../../../src/RTC/DtlsTransport.cpp:190:2: warning: control reaches end of non-void function [-Wreturn-type]
  190 |  }
      |  ^
../../../src/RTC/DtlsTransport.cpp: In static member function ‘static FBS::WebRtcTransport::DtlsRole RTC::DtlsTransport::RoleToFbs(RTC::DtlsTransport::Role)’:
../../../src/RTC/DtlsTransport.cpp:205:2: warning: control reaches end of non-void function [-Wreturn-type]
  205 |  }
      |  ^
../../../src/RTC/DtlsTransport.cpp: In static member function ‘static FBS::WebRtcTransport::DtlsState RTC::DtlsTransport::StateToFbs(RTC::DtlsTransport::DtlsState)’:
../../../src/RTC/DtlsTransport.cpp:226:2: warning: control reaches end of non-void function [-Wreturn-type]
  226 |  }
      |  ^
../../../src/RTC/DtlsTransport.cpp: In static member function ‘static RTC::DtlsTransport::FingerprintAlgorithm RTC::DtlsTransport::AlgorithmFromFbs(FBS::WebRtcTransport::FingerprintAlgorithm)’:
../../../src/RTC/DtlsTransport.cpp:248:2: warning: control reaches end of non-void function [-Wreturn-type]
  248 |  }
      |  ^
../../../src/RTC/DtlsTransport.cpp: In static member function ‘static FBS::WebRtcTransport::FingerprintAlgorithm RTC::DtlsTransport::AlgorithmToFbs(RTC::DtlsTransport::FingerprintAlgorithm)’:
../../../src/RTC/DtlsTransport.cpp:270:2: warning: control reaches end of non-void function [-Wreturn-type]
  270 |  }
      |  ^
../../../src/RTC/DtlsTransport.cpp: In member function ‘bool RTC::DtlsTransport::CheckRemoteFingerprint()’:
../../../src/RTC/DtlsTransport.cpp:1211:20: warning: ‘hashFunction’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1211 |   ret = X509_digest(certificate, hashFunction, binaryFingerprint, &size);
      |         ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1142/1172] Compiling C++ object mediasoup-worker.p/src_RTC_RtpDictionaries_RtpParameters.cpp.o
In file included from ../../../src/RTC/RtpDictionaries/RtpParameters.cpp:6:
../../../include/RTC/Codecs/Tools.hpp: In static member function ‘static bool RTC::Codecs::Tools::IsValidTypeForCodec(RTC::RtpParameters::Type, const RTC::RtpCodecMimeType&)’:
../../../include/RTC/Codecs/Tools.hpp:165:4: warning: control reaches end of non-void function [-Wreturn-type]
  165 |    }
      |    ^
../../../src/RTC/RtpDictionaries/RtpParameters.cpp: In static member function ‘static FBS::RtpParameters::Type RTC::RtpParameters::TypeToFbs(RTC::RtpParameters::Type)’:
../../../src/RTC/RtpDictionaries/RtpParameters.cpp:83:2: warning: control reaches end of non-void function [-Wreturn-type]
   83 |  }
      |  ^
```

Legitimate warnings are those:

```
[1227/1327] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelRequest.cpp.o
../../../src/Channel/ChannelRequest.cpp:105:50: warning: format specifies type 'unsigned int' but the argument has underlying type 'uint8_t' (aka 'unsigned char') [-Wformat]
                        MS_THROW_ERROR("unknown method '%" PRIu8 "'", this->method);
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
../../../include/MediaSoupErrors.hpp:32:48: note: expanded from macro 'MS_THROW_ERROR'
                MS_ERROR("throwing MediaSoupError: " desc, ##__VA_ARGS__); \
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
../../../include/Logger.hpp:397:122: note: expanded from macro 'MS_ERROR'
                        const int loggerWritten = std::snprintf(Logger::buffer, Logger::bufferSize, "E" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
                                                                                                                         ~~~~                 ^~~~~~~~~~~
../../../src/Channel/ChannelRequest.cpp:105:50: warning: format specifies type 'unsigned int' but the argument has underlying type 'uint8_t' (aka 'unsigned char') [-Wformat]
                        MS_THROW_ERROR("unknown method '%" PRIu8 "'", this->method);
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~
../../../include/MediaSoupErrors.hpp:33:77: note: expanded from macro 'MS_THROW_ERROR'
                std::snprintf(MediaSoupError::buffer, MediaSoupError::bufferSize, desc, ##__VA_ARGS__); \
                                                                                  ~~~~    ^~~~~~~~~~~
2 warnings generated.
[1229/1327] Compiling C++ object mediasoup-worker.p/src_Channel_ChannelNotification.cpp.o
../../../src/Channel/ChannelNotification.cpp:33:49: warning: format specifies type 'unsigned int' but the argument has underlying type 'uint8_t' (aka 'unsigned char') [-Wformat]
                        MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
../../../include/MediaSoupErrors.hpp:32:48: note: expanded from macro 'MS_THROW_ERROR'
                MS_ERROR("throwing MediaSoupError: " desc, ##__VA_ARGS__); \
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
../../../include/Logger.hpp:397:122: note: expanded from macro 'MS_ERROR'
                        const int loggerWritten = std::snprintf(Logger::buffer, Logger::bufferSize, "E" _MS_LOG_STR_DESC desc, _MS_LOG_ARG, ##__VA_ARGS__); \
                                                                                                                         ~~~~                 ^~~~~~~~~~~
../../../src/Channel/ChannelNotification.cpp:33:49: warning: format specifies type 'unsigned int' but the argument has underlying type 'uint8_t' (aka 'unsigned char') [-Wformat]
                        MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
../../../include/MediaSoupErrors.hpp:33:77: note: expanded from macro 'MS_THROW_ERROR'
                std::snprintf(MediaSoupError::buffer, MediaSoupError::bufferSize, desc, ##__VA_ARGS__); \
                                                                                  ~~~~    ^~~~~~~~~~~
2 warnings generated.
```

```
../../../src/Channel/ChannelNotification.cpp:33:49: warning: format specifies type 'unsigned int' but the argument has
underlying type 'uint8_t' (aka 'unsigned char') [-Wformat]
                        MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
```

This warning doesn't make any sense. I mean... format does NOT specify "unsigned int". It explicitly specified UINT8 (`PRIu8`), which I'd say it MATCHES the `uint8_t` type of the argument!!! WTH?

--- a/worker/src/Channel/ChannelNotification.cpp
+++ b/worker/src/Channel/ChannelNotification.cpp
@@ -30,12 +30,16 @@ namespace Channel
 		auto eventCStrIt = ChannelNotification::event2String.find(this->event);
 
 		if (eventCStrIt == ChannelNotification::event2String.end())
-			MS_THROW_ERROR("unknown event '%" PRIu8 "'", this->event);
+		{
+			MS_THROW_ERROR("unknown event '%" PRIu8 "'", static_cast<uint8_t>(this->event));
+		}
 
 		this->eventCStr = eventCStrIt->second;
 
 		// Handler ID is optional.
 		if (flatbuffers::IsFieldPresent(this->data, FBS::Notification::Notification::VT_HANDLERID))
+		{
 			this->handlerId = this->data->handlerId()->str();
+		}
 	}
 } // namespace Channel

--- a/worker/src/Channel/ChannelRequest.cpp
+++ b/worker/src/Channel/ChannelRequest.cpp
@@ -102,14 +102,16 @@ namespace Channel
 		{
 			Error("unknown method");
 
-			MS_THROW_ERROR("unknown method '%" PRIu8 "'", this->method);
+			MS_THROW_ERROR("unknown method '%" PRIu8 "'", static_cast<uint8_t>(this->method));
 		}
 
 		this->methodCStr = methodCStrIt->second;
 
 		// Handler ID is optional.
 		if (flatbuffers::IsFieldPresent(this->data, FBS::Request::Request::VT_HANDLERID))
+		{
 			this->handlerId = this->data->handlerId()->str();
+		}
 	}
 
 	void ChannelRequest::Accept()

